stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -107,17 +108,54 @@ private static String removeSuffix(String str, String[] suffixesToRemove) {
     Util.throwIfNull(handler);
     SQLiteDatabase database = openDatabase(databaseName);
     try {
-      Cursor cursor = database.rawQuery(query, null);
-      try {
-        return handler.handleResult(cursor);
-      } finally {
-        cursor.close();
+      String trimmedSql = query.trim();
```

Comment:
I think this can be done in a clearer way:
```java
String command = getFirstWord(query);
if ("UPDATE".equalsIgnoreCase(command)) { ... }

private static String getFirstWord(String s) {
  s.trim();
  int firstSpace = s.indexOf(' ');
  return firstSpace >= 0 ? s.substring(0, firstSpace) : s;
}
```